### ISSUE TITLE
Feature: Standardize App Name Across Platforms

### ISSUE THREAD

**Issue by jiongxuan** [johannesjo/super-productivity#3576]
## Problem Statement

The application name is displayed inconsistently across different platforms, which may affect brand recognition. For example, on Mac, the name appears as superProductivity, on Android it shows as SupProd, and on Windows and Ubuntu (as seen in screenshots) it displays as Super Productivity. This inconsistency could potentially hinder branding efforts.

iOS: 
![image](https://github.com/user-attachments/assets/ec875406-51ca-46bd-9f03-84120db0c972)

Android:
![image](https://github.com/user-attachments/assets/94870853-10f6-4334-8251-31288218d8c3)

## :grey_question: Possible Solution

I suggest unifying the application name across all platforms as Super Productivity. For Android, if there are concerns about the name being too long, a compromise could be to use Super Prod, which would still retain brand recognition without causing truncation issues.

Pixel: 
![image](https://github.com/user-attachments/assets/7560e5f2-07dd-4c10-95da-6afd41d3d552)

## :arrow_heading_up: Describe alternatives you've considered

An alternative could be to simplify the name to under 10 characters to avoid truncation, but this might not fully align with the brand’s established presence as Super Productivity. Another option is to use Super Prod on Android as a middle ground.

Which option do you think is better? If needed, I’m available to help make the changes.

**Comment by johannesjo** [comment]
I think we can close this, right @jiongxuan ?

**Comment by beerkumquatpome** [comment]
Please change the window class name to Super Productivity too (It's superProductivity)

**Comment by johannesjo** [comment]
I don't think this is possible without being confusing and inconvenient to existing users ( app data might be lost).
